cortex-ar/Cargo.toml

```diff
@@ -28,7 +28,7 @@ version = "0.1.0"
 arbitrary-int = "1.3.0"
 bitbybit = "1.3.3"
 num_enum = { version = "0.7", default-features = false }
-critical-section = {version = "1.2.0", features = ["restore-state-bool"], optional = true}
+critical-section = {version = "1.2.0", features = ["restore-state-u8"], optional = true}
```

Comment:
what about selecting bool for singel core, and u8 for multi core via the features table?

Reply:
They are the same size (1) so I think I'm OK with stuffing a boolean into a u8 for the single-core case.